File: modules/cli/src/test/resources/golden/expected/entities/Artikel.kt
package com.breuninger.entdecken.domain.model

import java.time.Instant
import java.util.Locale

data class Artikel(
    val id: ArtikelId,
    val ansichtenV2: Map<Int, String>,
    val beschreibungen: Map<Locale, Beschreibungen>,
    val farbe: Farbe?,
    val groesse: Groesse?,
    val startzeitpunktV2: Instant?,
    val endzeitpunktV2: Instant?,
    val neuBisDatum: Map<Land, Instant>,
    val firstSetActive: Map<Land, Instant>,
    val mengeV2: MengeV2?,
    val materialangaben: Materialangaben?,
    val attributeV2: List<AttributV2>,
    val vertriebsinfos: List<Vertriebsinfo>
)

File: modules/cli/src/test/resources/golden/input/entities/Artikel.scala
package com.breuninger.entdecken.domain.model

import java.time.Instant
import java.util.Locale

case class Artikel(
    id: ArtikelId,
    ansichtenV2: Map[Int, String],
    beschreibungen: Map[Locale, Beschreibungen],
    farbe: Option[Farbe],
    groesse: Option[Groesse],
    startzeitpunktV2: Option[Instant],
    endzeitpunktV2: Option[Instant],
    neuBisDatum: Map[Land, Instant],
    firstSetActive: Map[Land, Instant],
    mengeV2: Option[MengeV2],
    materialangaben: Option[Materialangaben],
    attributeV2: List[AttributV2],
    vertriebsinfos: List[Vertriebsinfo]
)
